core/templates/dev/head/domain/collection/CollectionValidationServiceSpec.js

```diff
@@ -297,17 +136,8 @@ describe('Collection validation service', function() {
     var node1 = _getCollectionNode('exp_id1');
     var node2 = _getCollectionNode('exp_id2');
 
-    node0.addPrerequisiteSkillId('skill0');
-    node0.addAcquiredSkillId('skill1');
-    node1.addPrerequisiteSkillId('skill1');
-    node2.addPrerequisiteSkillId('skill1');
-
     var issues = _findPublicValidationIssues();
     expect(issues).toEqual([
-      'There should be exactly 1 exploration initially available to the ' +
```

Comment:
Might want a test covering this error? (if 0 exps exist?)

Reply:
Actually, I now removed this issue from CollectionValidationService.js also, as this condition is the same as if no explorations are there in the collection, and that is already being tested.